src/python/module/z5py/attribute_manager.py

```diff
@@ -13,21 +13,21 @@ def __init__(self, path, is_zarr):
 
     def __getitem__(self, key):
 
```

Comment:
Is this for compatibility with the C++ layer? It's a `KeyError` in h5py, which is more in line with python conventions around item getting.